dist/common/scripts/scylla_kernel_check

```diff
@@ -31,12 +31,7 @@ if __name__ == '__main__':
         sys.exit(1)
 
     if not shutil.which('mkfs.xfs'):
-        if is_debian_variant():
-            run('apt-get install -y xfsprogs')
-        elif is_gentoo_variant():
-            run('emerge -uq sys-fs/xfsprogs')
-        elif is_redhat_variant():
-            run('yum install -y xfsprogs')
+        pkg_install('xfsprogs')
```

Comment:
I think this breaks gentoo.

Maybe we can do something like this:

    pkg_install('xfsprogs', {'gentoo': 'sys-fs/xfsprogs'})

To provide exceptional names when they aren't the same.

Reply:
Actually, `emerge -uq xfsprogs` works (well I see it start, I'm not going to wait for it to finish).

The idea still makes sense, but not needed here.

Reply:
I like this approach. I think we could (in a future patch) simplify the code by consolidating the different code paths to use this proposed variant of `pkg_install()` instead of also having OS-specific paths where we call `apt`, `emerge`, and so on.

Reply:
It's better to passing category name to emerge command that is true, but the command should work unless there are same name packages between different category.
If emerge detects multiple packages, it causes error like this:
```
6b5e41425118 / # emerge -uq mysql

 * IMPORTANT: 7 news items need reading for repository 'gentoo'.
 * Use eselect news read to view new items.



!!! The short ebuild name "mysql" is ambiguous. Please specify
!!! one of the following fully-qualified ebuild names instead:

    acct-group/mysql
    acct-user/mysql
    dev-db/mysql
    dev-haskell/mysql
    virtual/mysql
```
So current code should work in limited condition: it's only package name in all categories.
If there are two or more, then we have to pass package name with category.

But maybe it's not clean approach, we can have dict to resolve full package name for emerge, something like this `{'xfsprogs': 'sys-fs/xfsprogs'}`

Reply:
Yes, a side dict for translating gentoo package names is a good idea.